pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -43,6 +43,25 @@ pub mod pallet {
 	};
 	use sp_std::vec::Vec;
 
+	#[derive(Debug, Encode, Decode, MaxEncodedLen, TypeInfo)]
+	pub struct SeasonConfigs<Balance, BlockNumber> {
+		pub mint_available: bool,
+		pub mint_fees: Balance,
+		pub mint_cooldown: BlockNumber,
+	}
+
+	pub type SeasonConfigsOf<T> =
+		SeasonConfigs<BalanceOf<T>, <T as frame_system::Config>::BlockNumber>;
+
+	#[pallet::type_value]
+	pub fn DefaultConfigs<T: Config>() -> SeasonConfigsOf<T> {
```

Comment:
Just for consistency:
```suggestion
	pub fn DefaultSeasonConfigs<T: Config>() -> SeasonConfigsOf<T> {
```